server/sonar-web/src/main/js/components/issue/components/IssueTitleBar.js

```diff
@@ -41,6 +44,24 @@ export default function IssueTitleBar(props: Props) {
   const { issue } = props;
   const hasSimilarIssuesFilter = props.onFilter != null;
 
+  const locationsCount =
+    issue.secondaryLocations.length +
+    issue.flows.reduce((sum, locations) => sum + locations.length, 0);
+
+  const locationsBadge = (
+    <Tooltip
+      overlay={translateWithParameters(
+        'issue.this_issue_involves_x_code_locations',
+        formatMeasure(locationsCount)
+      )}
+      placement="left">
+      <LocationIndex>{locationsCount}</LocationIndex>
+    </Tooltip>
+  );
+
+  // dirty trick :(
+  const onIssuesPage = document.getElementById('issues-page') != null;
```

Comment:
Is this used to hide the label with the number of flows when we are not on the issues list ?
Because I still have the label when I click on the issue and I'm on the source viewer, I'm not sure this is wanted or not ?